test/arm/neon/st1_lane.c

```diff
@@ -0,0 +1,1244 @@
+#define SIMDE_TEST_ARM_NEON_INSN st1_lane
+
+#include "test-neon.h"
+#include "../../../simde/arm/neon/st1_lane.h"
+
+static int
+test_simde_vst1_lane_f32 (SIMDE_MUNIT_TEST_ARGS) {
+#if 1
```

Comment:
leftover from debugging?

Reply:
Nah, we (Evan, rosbif, and I) decided to leave the test generation code in there in case we need to revisit it later.

Reply:
Yeah, in the NEON implementations the test files are small enough that it's not a problem.  For x86 we still delete the test generation code, though, since those are massive.

It also helps that we have [a separate repository](https://github.com/simd-everywhere/simde-no-tests) without the tests, so the bloat shouldn't cause problems for projects embedding SIMDe.